rktl_sim/config/simulation.yaml

```diff
@@ -1,4 +1,18 @@
 rate: 10
 spawn_height: 0.06
-init:
-  speed: 1.0
+
+car:
+  # init_pose:
+  #   pos: [0.5, 0.0, 0.06]
+  sensor_noise:
+    pos: [0.01, 0.01, 0.00]
+    orient: [0.0, 0.0, 0.01]
+
+ball:
+  init_speed: 0.0
+  # init_pose:
+  #   pos: [0.0, 0.5, 0.06]
+  #   orient: [0.0, 0.0, 0.0]
```

Comment:
are these parameters valid / in use? If they are commented for a reason, pls explain why in the file.

Reply:
Valid parameters, I left it there as an example